runtime/doc/autocmd.txt

```diff
@@ -1053,7 +1054,9 @@ WinLeave			Before leaving a window.  If the window to be
 				executes the BufLeave autocommands before the
 				WinLeave autocommands (but not for ":new").
 				Not used for ":qa" or ":q" when exiting Vim.
-
+							*WinClosed*
+WinClosed			After closing a window.  Just before freeing
+				window's resources.
```

Comment:
Recursion is ignored.

Users don't care about "resources" and it's just confusing to mention that, it is never mentioned anywhere else in the help docs.